src/geventhttpclient/useragent.py

```diff
@@ -59,6 +59,8 @@ class CompatRequest(object):
     """ urllib / cookielib compatible request class.
         See also: http://docs.python.org/library/cookielib.html
     """
+    unverifiable = False
```

Comment:
The variable is unused. Did you plan to use it in `is_unverifiable`?

Reply:
It's used in stdlib's `http.cookiejar.DefaultCookiePolicy.set_ok_verifiability()`. This PR fixes so that one can actually use `UserAgent(cookiejar=CookieJar())` on Python >=3.3.

I hard coded it to False since that's what `is_verifiable()` (which is used in older python versions) does, but I guess one could also make it a property that calls `is_verifiable()` (or the other way around).

Reply:
Do you want me to update the PR to call `is_verifiable()` from a `verifiable` property?